main/lsp/ExtractVariable.cc

```diff
@@ -127,6 +128,7 @@ class LocSearchWalk {
     // For example, the RHS of the ClassDef doesn't have an ExpressionPtr with a Loc,
     // but enclosingScopeLoc will be a Loc that represents the body of the ClassDef RHS
     // (excluding things like the class name, superclass, and class/end keywords).
+    // TODO(neil): can we get rid of enclosingScopeLoc?
```

Comment:
Any more you want to investigate here before landing?

Reply:
No, this was just a idea I had when looking at the code, but I didn't want to spend the ~20 minutes  for trying to refactor and convince myself that the refactor is safe right now, so I decided to add a TODO for the future.